tribe-ext-relabeler/index.php

```diff
@@ -0,0 +1,350 @@
+				'default'         => esc_attr__( 'Venue', 'the-events-calendar' ),
+				'tooltip'         => esc_html__( 'Singular label for Venues.', 'tribe-extension' ),
+				'validation_type' => 'html',
+			),
+			'label_venue_single_lowercase' => array(
+				'type'            => 'text',
+				'label'           => esc_html__( 'venue', 'the-events-calendar' ),
+				'default'         => esc_attr__( 'venue', 'the-events-calendar' ),
+				'tooltip'         => esc_html__( 'Lowercase singular label for Venues.', 'tribe-extension' ),
+				'validation_type' => 'html',
+			),
+			'label_venue_plural' => array(
+				'type'            => 'text',
+				'label'           => esc_html__( 'Venues', 'the-events-calendar' ),
+				'default'         => esc_attr__( 'Venues', 'the-events-calendar' ),
+				'tooltip'         => esc_html__( 'Plural label for Venues.', 'tribe-extension' ),
+				'validation_type' => 'html',
+			),
+			'label_venue_plural_lowercase' => array(
+				'type'            => 'text',
+				'label'           => esc_html__( 'venues', 'the-events-calendar' ),
+				'default'         => esc_attr__( 'venues', 'the-events-calendar' ),
+				'tooltip'         => esc_html__( 'Lowercase plural label for Venues.', 'tribe-extension' ),
+				'validation_type' => 'html',
+			),
+			'label_organizer_single' => array(
+				'type'            => 'text',
+				'label'           => esc_html__( 'Organizer', 'the-events-calendar' ),
+				'default'         => esc_attr__( 'Organizer', 'the-events-calendar' ),
+				'tooltip'         => esc_html__( 'Singular label for Organizers.', 'tribe-extension' ),
+				'validation_type' => 'html',
+			),
+			'label_organizer_single_lowercase' => array(
+				'type'            => 'text',
+				'label'           => esc_html__( 'organizer', 'the-events-calendar' ),
+				'default'         => esc_attr__( 'organizer', 'the-events-calendar' ),
+				'tooltip'         => esc_html__( 'Lowercase singular label for Organizers.', 'tribe-extension' ),
+				'validation_type' => 'html',
+			),
+			'label_organizer_plural' => array(
+				'type'            => 'text',
+				'label'           => esc_html__( 'Organizers', 'the-events-calendar' ),
+				'default'         => esc_attr__( 'Organizers', 'the-events-calendar' ),
+				'tooltip'         => esc_html__( 'Plural label for Organizers.', 'tribe-extension' ),
+				'validation_type' => 'html',
+			),
+			'label_organizer_plural_lowercase' => array(
+				'type'            => 'text',
+				'label'           => esc_html__( 'organizers', 'the-events-calendar' ),
+				'default'         => esc_attr__( 'organizers', 'the-events-calendar' ),
+				'tooltip'         => esc_html__( 'Lowercase plural label for Organizers.', 'tribe-extension' ),
+				'validation_type' => 'html',
+			),
+		);
+
+		$setting_helper->add_fields(
+			$fields,
+			'display',
+			'tribeEventsDateFormatSettingsTitle',
+			true
+		);
+	}
+
+	/**
+	 * Get an HTML link to the General settings tab
+	 *
+	 * @return string <a> link to the general settings tab
```

Comment:
Nitpick: 
> HTML elements and tags should be written as “audio element” or “link tag”.
From [here](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#docblock-formatting).
I would recommend `"link tag"` in the comment instead of `<a>`